lib/couchbase-orm/utilities/query_helper.rb

```diff
@@ -132,8 +132,11 @@ def serialize_value(key, value_before_type_cast)
             end
 
             def quote(value)
-                if [String, DateTime, Date, Time].any? { |clazz| value.is_a?(clazz) }
+                if [String, Date, Time].any? { |clazz| value.is_a?(clazz) }
                     "'#{N1ql.sanitize(value)}'"
+                elsif value.is_a? DateTime
+                    formatedDate = DateTime.serialize(value)
```

Comment:
Would it be possible to verify the time format in the query? To guarantee that the actual n1ql has a `iso8601` format.

Reply:
I will directly apply `value&.iso8601(@precision)`